### PR TITLE
Add filter, map, flatMap method

### PR DESCRIPTION
**Pull request by hhp227**
Add filter, map, flatMap method

### CODE REVIEW

File: Sources/Paging/Pager/PageFetcher.swift [hunk 1]
```diff
@@ -13,15 +13,20 @@ internal class PageFetcher<Key: Equatable, Value: Any> {
...
+    private let retryEvents = ConflatedEventBus<Void>()
+
     var publisher: AnyPublisher<PagingData<Value>, Never> {
         get {
+            let remoteMediatorAccessor = remoteMediator != nil ? remoteMediatorAccessor(delegate: remoteMediator!) : nil
             let initialValue: GenerationInfo<Key, Value>? = nil
             return refreshEvents.publisher
-                .prepend(true)
-                .scan(initialValue) { previousGeneration, _ in
+                .prepend(remoteMediatorAccessor?.initialize() == .LAUNCH_INITIAL_REFRESH)
```
Comment (greenthings) [inline Sources/Paging/Pager/PageFetcher.swift]: Please, make this lowercase for enum case custom.

File: Sources/Paging/PageEvent.swift [hunk 2]
```diff
@@ -35,6 +33,73 @@ open class PageEvent<T: Any> {
         
         var mediatorLoadStates: LoadStates? = nil
         
+        private func mapPages<R: Any>(_ transform: @escaping (TransformablePage<T>) -> TransformablePage<R>) -> Insert<R> {
```
Comment (greenthings) [inline Sources/Paging/PageEvent.swift:36]: To improve code readability, what do you think about changing 'R' to 'Result'?

**Review by greenthings — CHANGES_REQUESTED** [review]
It was good to convert the enum case from lowercase to uppercase.

File: Sources/Paging/PageEvent.swift (hunk 2 above)
Comment (hhp227study) [reply Sources/Paging/PageEvent.swift:36]: No, R means Result

Comment (hhp227) [reply Sources/Paging/PageEvent.swift:36]: No, R means Result

Comment (greenthings) [reply Sources/Paging/PageEvent.swift:36]: OK, got it, Do what you want

File: Sources/Paging/Pager/PageFetcher.swift (hunk 1 above)
Comment (hhp227study) [reply Sources/Paging/Pager/PageFetcher.swift]: change done

**Pull request merged**